images-list

```diff
@@ -3,4 +3,12 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+grafana/grafana rancher/grafana-grafana 7.1.5
+istio/kubectl rancher/istio-kubectl 1.5.10
 library/busybox rancher/library-busybox 1.31.1
+library/nginx rancher/nginx 1.19.2-alpine
+prom/alertmanager rancher/prom-alertmanager v0.21.0
+prom/node-exporter rancher/prom-node-exporter v1.0.1
+prom/prometheus rancher/prom-prometheus v2.21.0
```

Comment:
The current master chart uses version 2.18.2 https://github.com/prometheus-community/helm-charts/blob/4d780c4496a7df089dfde11c8731f26dc51c4a1f/charts/kube-prometheus-stack/values.yaml#L1617-L1618, which is also what we need for https://github.com/rancher/charts/pull/641.

Reply:
Looks like needed in v1, so likely need both.

Reply:
@cbron the change isn't merged into v1 yet, so I left the same comment on that unmerged PR. https://github.com/rancher/system-charts/pull/334#discussion_r489709105

Reply:
@aiyengar2 @PennyScissors The v1 monitoring and v2 monitoring tag versions can/should stay completely separate. They can use different tags and thats ok. What I meant above was that if we are changing the _name_ of the tag from one to another with this new framework that might affect both of them.

Reply:
@cbron not sure I agree that the tag versions should stay separate. 

It is true that Monitoring V1 / Monitoring V2 should have no connection to each other; however, both charts are still based off of the upstream chart at [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/tree/4d780c4496a7df089dfde11c8731f26dc51c4a1f/charts/kube-prometheus-stack), so to avoid any weird / extraneous issues that we would have to resolve (as opposed to upstream) it would be better if we can keep Monitoring V1 in sync with the images that are used by `kube-prometheus-stack`. 

If there's a specific requirement for Rancher for a higher version of Prometheus due to some specific feature request that only a higher version of Prometheus can support, it makes sense to have Monitoring V1 / V2 use different tag versions, but I don't think we've ever encountered such a use case.

Reply:
Ok, I'll downgrade to 2.18.2

Reply:
We can stay on same version for now, that will let customers migrate easily between the two with same configuration. But if we are doing security updates in future we don't need to wait for other half to keep them in sync.